.github/workflows/update-wpt.yml

```diff
@@ -0,0 +1,82 @@
+name: WPT update
+
+on:
+  schedule:
+    # Run once a week at 12:00 AM UTC on Sunday.
+    - cron: 0 0 * * *
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  wpt-subsystem-update:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        subsystem:
+          - eventsource
+          - fetch
+          - websockets
+
+    steps:
+      - uses: actions/checkout@v4
```

Comment:
For security reasons, I believe all of these should be referenced by hash, not tag.

Reply:
applied, thanks